LICENSE-binary

```diff
@@ -257,12 +258,16 @@ org.apache.ratis:ratis-netty
 org.apache.ratis:ratis-proto
 org.apache.ratis:ratis-server
 org.apache.ratis:ratis-server-api
+org.apache.ratis:ratis-shell
 org.apache.ratis:ratis-thirdparty-misc
+org.javassist:javassist
+org.reflections:reflections
 org.roaringbitmap:RoaringBitmap
 org.roaringbitmap:shims
 org.scala-lang:scala-library
 org.scala-lang:scala-reflect
 org.slf4j:jcl-over-slf4j
+org.slf4j:slf4j-simple
```

Comment:
we should exclude it since we use log4j2

Reply:
> we should exclude it since we use log4j2

Done